app/models/mixins/authentication_mixin.rb

```diff
@@ -273,6 +273,27 @@ def authentication_check(*args)
     return status == :valid, details
   end
 
+  def authentication_check_queue(*args)
```

Comment:
Can you use the nearly identical `authentication_check_types_queue`?

It's what we call from the schedule worker on a schedule for each model that includes the authentication mixin.

https://github.com/ManageIQ/manageiq/blob/fa42171d9cac349eea580a96c134384de949a5d2/app/models/mixins/authentication_mixin.rb#L10-L13